src/styles/styles.js

```diff
@@ -2878,6 +2878,48 @@ const styles = {
         paddingTop: 80,
         paddingBottom: 45,
     },
+
+    footerWrapper: {
+        backgroundColor: themeColors.midtone,
+        fontSize: variables.fontSizeNormal,
+        paddingTop: 64,
+        paddingLeft: 17,
+    },
+
+    footerColumnsContainer: {
+        maxWidth: 1100,
+        flex: 1,
+        flexWrap: 'wrap',
+        marginBottom: 40,
+    },
+
+    footerColumn: {
+        padding: 15,
+    },
+
+    footerTitle: {
+        fontSize: variables.fontSizeLarge,
+        color: themeColors.success,
+        marginBottom: 16,
+    },
+
+    footerRow: {
+        paddingVertical: 4,
+        marginBottom: 8,
+        color: themeColors.textLight,
+        fontSize: variables.fontSizeMedium,
+    },
+
+    footerLargeScreenAlign: {
+        minWidth: '80%',
+        alignSelf: 'center',
+    },
+
+    footerTermsContainer: {
+        marginVertical: 16,
+        maxWidth: 200,
```

Comment:
What happens when we get rid of the maxWidth?

Reply:
Here's what it looks like: I think I just need to handle the overflow in that case

https://user-images.githubusercontent.com/38015950/218580405-efbd28f2-28a7-4ad1-a2ad-84b979b58232.mov

Reply:
Cool, I think we can get rid of it. And then I also answered your comment that perhaps we want to break it up into two paragraphs.